bot/converters.py

```diff
@@ -163,18 +155,18 @@ def is_number(value: str) -> bool:
                         "Rejecting the request.")
             raise BadArgument("Tag names should not be empty, or filled with whitespace.")
 
-        # The tag name is a number of some kind, we don't allow that.
-        elif is_number(tag_name):
-            log.warning(f"{ctx.author} tried to create a tag with a digit as its name. "
-                        "Rejecting the request.")
-            raise BadArgument("Tag names can't be numbers.")
-
         # The tag name is longer than 127 characters.
         elif len(tag_name) > 127:
             log.warning(f"{ctx.author} tried to request a tag name with over 127 characters. "
                         "Rejecting the request.")
             raise BadArgument("Are you insane? That's way too long!")
 
+        # The tag name is ascii but does not contain any letters.
+        elif not any(character.isalpha() for character in tag_name):
+            log.warning(f"{ctx.author} tried to request a tag name without letters. "
+                        "Rejecting the request.")
```

Comment:
Especially now that we use Sentry, these shouldn't warnings. In fact, with #757, there'll be no need to log this as the exception will already get logged in the command error handler. These log messages can be removed from this converter.